docs/USAGE-GUIDE.md

```diff
@@ -413,17 +413,21 @@ The following chart maps the security policy version to protocol version and cip
 |          "20190801"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |   X   |                  |
 |          "20190802"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |   X   |                  |
 |          "20200207"                          |       |   X    |    X   |    X   |    X    |    X    |          X        |       |    X    |      |     |     |       |                  |
+|          "20230317"                          |       |        |        |    X   |    X    |         |                   |   X   |    X    |      |     |     |   X   |                  |
```

Comment:
This security policy you added does support AES-CBC

Reply:
🤦‍♀️That's what I get for waffling ;)